test_server/server.go

```diff
@@ -309,6 +309,7 @@ func (s *TestServer) handleV2(w http.ResponseWriter, r *http.Request) {
 		route := "/" + SYNQ_ROUTE
 		switch r.URL.Path {
 		case route + "/videos/" + V2_VIDEO_ID,
+			route + "/videos/" + V2_VIDEO_ID2,
```

Comment:
I believe some tests expect VIDEO_ID2 to return error, so if this is added, we may need to fix other tests.  Just a FYI

Reply:
I checked aerico, ifrit, pixie, unicorn and they are not using it. I'll check which other repos are using synq-golang.

Reply:
ok, we can just fix it as needed